<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="21448" started-at="2020-02-01T15:48:36Z" finished-at="2020-02-01T15:48:57Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="21448" started-at="2020-02-01T15:48:36Z" finished-at="2020-02-01T15:48:57Z">
      <class name="MavenTutorial.MavenTutorial.Test4">
        <test-method status="PASS" signature="testDocker()[pri:0, instance:MavenTutorial.MavenTutorial.Test4@2ed94a8b]" name="testDocker" duration-ms="9722" started-at="2020-02-01T21:18:48Z" description="This test is used for docker run" finished-at="2020-02-01T21:18:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDocker -->
      </class> <!-- MavenTutorial.MavenTutorial.Test4 -->
      <class name="MavenTutorial.MavenTutorial.Test1">
        <test-method status="PASS" signature="mavenBasic()[pri:0, instance:MavenTutorial.MavenTutorial.Test1@1554909b]" name="mavenBasic" duration-ms="24" started-at="2020-02-01T21:18:36Z" description="Mavn profile and parameters" finished-at="2020-02-01T21:18:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mavenBasic -->
      </class> <!-- MavenTutorial.MavenTutorial.Test1 -->
      <class name="MavenTutorial.MavenTutorial.Test3">
        <test-method status="FAIL" signature="testDocker()[pri:0, instance:MavenTutorial.MavenTutorial.Test3@73f792cf]" name="testDocker" duration-ms="6142" started-at="2020-02-01T21:18:41Z" description="This test is used for docker run" finished-at="2020-02-01T21:18:48Z">
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:48)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertFalse(Assert.java:34)
	at junit.framework.Assert.assertFalse(Assert.java:41)
	at MavenTutorial.MavenTutorial.Test3.testDocker(Test3.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDocker -->
      </class> <!-- MavenTutorial.MavenTutorial.Test3 -->
      <class name="MavenTutorial.MavenTutorial.Test2">
        <test-method status="PASS" signature="testDocker()[pri:0, instance:MavenTutorial.MavenTutorial.Test2@38082d64]" name="testDocker" duration-ms="5545" started-at="2020-02-01T21:18:36Z" description="This test is used for docker run" finished-at="2020-02-01T21:18:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDocker -->
      </class> <!-- MavenTutorial.MavenTutorial.Test2 -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
